### DISCUSSION TITLE
I hope that when Leaflet1.9 popups contain images, the border of the popus can be increased

### DISCUSSION THREAD

**Discussion by ooof** [Leaflet/Leaflet#8822] [Q&A]
I've just started using Leaflet and I think it's great, but I've run into a problem and have searched but not solved it.

I am using Leaflet version 1.9 and in the popup I have added an image and if I set the image width to greater than 300 the image will go beyond the border of the popup, see the image below：

![2023-01-31 09:20:45](https://user-images.githubusercontent.com/392785/215690759-0a97aa56-2e53-44bf-ba12-c7a70301ab17.png)

Related html:

`<style>
.img-in-popup { width:500px; }
</style>

const marker = L.marker([36.153994, 111.490025]).bindPopup('<img class="img-in-popup"" src="./img/P30117-105203.jpg" />').addTo(map);`

I wish the border of the image would follow the image to make it bigger.

I see that this is a situation that has existed for a long time and hope to find a solution.

thanks!

**Comment by IvanSanchez** [comment]
https://leafletjs.com/reference.html#popup-maxwidth

**Comment by ooof** [comment]
I have tried the following codes and they work  :

const popupdd = L.popup()
	.setLatLng([36.154, 111.495])
	.setContent('I am a standalone popup.')
	.openOn(map);


But I don't know how to add "maxWidth" to it, can anyone tell me?

Thank you very much!

**Comment by anton-seaice** [comment]
I think you want to set the maxWidth option for your popup. Which could be done like this:

`L.marker([36.153994, 111.490025]).bindPopup('<img class="img-in-popup"" src="./img/P30117-105203.jpg" />', {maxWidth:600}).addTo(map);`

You could also mess with the css for .leaflet-popup-content-wrapper but this would impact all popups.

**Comment by ooof** [comment]
@anton-seaice  I managed to achieve this effect, thank you very much!

Good luck!
